shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/repositories/SettingsRepository.kt

```diff
@@ -35,10 +36,9 @@ class SettingsRepository : ISettingsRepository, KoinComponent {
 }
 
 enum class Settings(val dataStoreKey: Preferences.Key<Boolean>) {
-    Map(booleanPreferencesKey("map_debug")),
+    DevDebugMode(booleanPreferencesKey("dev_debug_mode")),
```

Comment:
A little bit breaking that hard line, I was wondering if we should have a "secret" command for enabling debug mode in prod, something like tapping the T logo on the more page 10 times in couple seconds, just because I assume we (or more likely other people in the department who don't have access to staging) will sometimes run into data issues when we're using the prod app and will want to have some way to see that information.

Reply:
Oh interesting! Will bring it up to product I do think we'd want an easy "turn off" button in that case. Will definitely leave out of this PR and can address in follow up PR or split a follow-up ticket for it.